filebeat/module/logstash/log/test/logstash-plain.log-expected.json

```diff
@@ -1,25 +1,25 @@
 [
     {
-        "@timestamp": "2017-10-23T14:20:12,046", 
-        "event.dataset": "log", 
-        "event.module": "logstash", 
-        "input.type": "log", 
-        "log.offset": 0, 
-        "logstash.log.level": "INFO", 
-        "logstash.log.message": "Initializing module {:module_name=>\"fb_apache\", :directory=>\"/usr/share/logstash/modules/fb_apache/configuration\"}", 
+        "@timestamp": "2017-10-23T14:20:12.046Z",
+        "event.dataset": "log",
```

Comment:
Seems like all the files you generated have that space diff. I wonder if it's your editor adjusting it or if this is depends on the OS perhaps. CI is green so ...